src/fhe.rs

```diff
@@ -60,6 +61,27 @@ impl FheApp {
         }
     }
 
+    /// Generate keys for an FHE application.
+    pub fn generate_keys(&self) -> Result<(PublicKey, PrivateKey), RuntimeError> {
+        let (public_key, private_key) = self.runtime.generate_keys()?;
+
+        Ok((public_key, private_key))
+    }
+
+    /// Generate keys for an FHE application without the galois keys, which
+    /// enables a significant reduction in key size if you do not plan to
+    /// perform the rotations that the Galois keys provide.
+    pub fn generate_keys_without_galois(&self) -> Result<(PublicKey, PrivateKey), RuntimeError> {
```

Comment:
I think this is more confusing. For example, if we updated the dependency in `sunscreen_rpc`, everything would compile but we'd be sending 13MB keys all of a sudden. And since this is living in `testnet::one`, it doesn't seem like we need both. Can we remove this?

Or, if there is a legitimate reason to generate with the galois key, can you rename that one instead to `generate_keys_with_galois`?

Reply:
I suppose we have no need for the generate with the galois keys at the moment so I will remove it and just have `generate_keys`.